src/core/PWSprefs.cpp

```diff
@@ -132,6 +132,8 @@ const PWSprefs::boolPref PWSprefs::m_bool_prefs[NumBoolPrefs] = {
   {_T("ListSortAscending"), true, ptApplication},           // application
   {_T("EnableWindowTransparency"), false, ptApplication},   // application
   {_T("ShowMenuSeparator"), true, ptApplication},           // application
+  {_T("OptimizedCellSize"), false, ptApplication},          // application
+  {_T("ToolbarShowText"), true, ptApplication},             // application
```

Comment:
I would suggest not to surprise the user with a change in the user interface by default and thus set the default value for "ToolbarShowText" to "false".

Reply:
Adapted to false as demanded.